api-specifications.md

```diff
@@ -48,13 +48,24 @@ Each file and directory tree will be transferred individually.
 
 ---
 
+## Authentication
+
+### vserver
```

Comment:
I wonder if vserver should be documented here. Maybe we should have a separate document for vserver, as no one should be using it in production.

Reply:
I will say since we are currently developing almost all the backend on vserver, separating the API docs will be too costly (we need to maintain two almost-the-same version). I think just adding a note saying "don't use it" is good enough.

Reply:
Then we can remove it after vserver is abandoned.

Reply:
The documents will not be similar. api-specifications.md will have the specs for ngshare. Nothing about vserver would be on it. If you want to have documentation on vserver, I would like it to be in a separate document.

In summary, avoid putting vserver comments in the specs.